multisig/src/main.rs

```diff
@@ -40,17 +42,18 @@ struct Multisig {
 }
 
 impl Multisig {
-    pub fn new(_ctx: &Context, owners: Set<Address>, required: u32) -> Self {
+    pub fn new(_ctx: &Context, addresses: Vec<String>, required: u32) -> Self {
         Self {
             transactions: Map::new(),
-            owners: owners,
+            owners: Set::from_iter(addresses.iter().map(|a| Address::from_str(a).unwrap())),
             required: required,
```

Comment:
```suggestion
            required,
```
please run `cargo clippy` on your code and add it to CI